extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -116,98 +114,18 @@ public static void startServerAfterFailedStart() {
                 public Integer get() {
                     return ProcessorInfo.availableProcessors() * 2; //this is dev mode, so the number of IO threads not always being 100% correct does not really matter in this case
                 }
-            });
+            }, null);
 
         } catch (Exception e) {
             throw new RuntimeException(e);
         }
     }
 
-    public RuntimeValue<Router> initializeRouter(final RuntimeValue<Vertx> vertxRuntimeValue,
```

Comment:
Why has all this code been moved around? It makes it much harder to tell what has actually changed when reviewing.

Reply:
sure, fixed now, endup applying the reformat for the entire class.